draft-ietf-webtrans-http2.md

```diff
@@ -396,6 +393,16 @@ WT_STREAM Frame {
 ~~~
 {: #fig-wt_stream title="WT_STREAM Frame Format"}
 
+WT_STREAM frames contain the following fields:
+
+Stream ID:
+: The stream ID for the stream.
+
+Stream Data:
+: Zero or more bytes of data for the stream.  Empty WT_STREAM frames MUST NOT be
+used unless they open or close a stream; an endpoint MAY treat an empty
+WT_STREAM frame that neither starts nor ends a stream as a session error.
```

Comment:
Ah, nice. We definitely need the ability to open/close without sending, but no need to leave that wide open.